packages/audioplayers/tizen/src/audioplayers_tizen_plugin.cc

```diff
@@ -90,89 +90,68 @@ class AudioplayersTizenPlugin : public flutter::Plugin {
       return;
     }
-    GetValueFromEncodableMap(arguments, "mode", mode);
-
-    AudioPlayer *player = GetAudioPlayer(player_id, mode);
+    AudioPlayer *player = GetAudioPlayer(player_id);
 
     try {
       const std::string &method_name = method_call.method_name();
-      if (method_name == "play") {
-        double volume = 0.0;
-        if (GetValueFromEncodableMap(arguments, "volume", volume)) {
-          player->SetVolume(volume);
-        }
-
-        std::string url;
-        if (GetValueFromEncodableMap(arguments, "url", url)) {
-          player->SetUrl(url);
-        }
-
-        player->Play();
-
-        int32_t position = 0;
-        if (GetValueFromEncodableMap(arguments, "position", position)) {
-          player->Seek(position);
-        }
-
-        result->Success(flutter::EncodableValue(1));
-      } else if (method_name == "playBytes") {
-        double volume = 0.0;
-        if (GetValueFromEncodableMap(arguments, "volume", volume)) {
-          player->SetVolume(volume);
-        }
-
-        std::vector<uint8_t> bytes;
-        if (GetValueFromEncodableMap(arguments, "bytes", bytes)) {
-          player->SetDataSource(bytes);
-        }
-
-        player->Play();
-
-        int32_t position = 0;
-        if (GetValueFromEncodableMap(arguments, "position", position)) {
-          player->Seek(position);
-        }
-
-        result->Success(flutter::EncodableValue(1));
+      if (method_name == "setSourceBytes") {
+        std::vector<uint8_t> bytes =
+            GetRequiredArg<std::vector<uint8_t>>(arguments, "bytes");
+        player->SetDataSource(bytes);
+        result->Success();
       } else if (method_name == "resume") {
         player->Play();
-        result->Success(flutter::EncodableValue(1));
+        result->Success();
       } else if (method_name == "pause") {
         player->Pause();
-        result->Success(flutter::EncodableValue(1));
+        result->Success();
       } else if (method_name == "stop") {
         player->Stop();
-        result->Success(flutter::EncodableValue(1));
+        result->Success();
       } else if (method_name == "release") {
         player->Release();
-        result->Success(flutter::EncodableValue(1));
+        result->Success();
       } else if (method_name == "seek") {
         player->Seek(GetRequiredArg<int32_t>(arguments, "position"));
-        result->Success(flutter::EncodableValue(1));
+        result->Success();
       } else if (method_name == "setVolume") {
         player->SetVolume(GetRequiredArg<double>(arguments, "volume"));
-        result->Success(flutter::EncodableValue(1));
-      } else if (method_name == "setUrl") {
-        player->SetUrl(GetRequiredArg<std::string>(arguments, "url"));
-        result->Success(flutter::EncodableValue(1));
+        result->Success();
+      } else if (method_name == "setSourceUrl") {
+        bool isLocal = false;
+        GetValueFromEncodableMap(arguments, "playerId", isLocal);
```

Comment:
`isLocal` should be `is_local`.

Also why does `playerId` map to a boolean value?

Reply:
Oh, this is a mistake. Fixed!